modules/migration-compatibility-guidelines.adoc

```diff
@@ -20,29 +20,32 @@ modern operator:: The {mtc-short} Operator designed for modern platforms.
 control cluster:: The cluster that runs the {mtc-short} controller and GUI.
 remote cluster:: A source or destination cluster for a migration that runs Velero. The Control Cluster communicates with Remote clusters via the Velero API to drive migrations.
 
-
-[cols="1,2,2", options="header"]
 .{mtc-short} compatibility: Migrating from a legacy platform
```

Comment:
The table is missing the `cols` attribute. See [link](https://docs.asciidoctor.org/asciidoc/latest/tables/build-a-basic-table/#create-a-table-with-two-columns-and-three-rows)

Reply:
Also consider adding [cols=<column_specifiers>,options="header,word-wrap",subs="+quotes,+attributes"], so the *Important* syntax resolves to bold highlighting.